### DISCUSSION TITLE
Ability to Read News Sites from bottom [Old News] To Top [Latest News]

### DISCUSSION THREAD

**Discussion by icemanmelb** [ken107/read-aloud#417] [Q&A]
I tend to use Text to Speech on a lot of news headline site and was wondering if Read-Aloud or any extension have the Ability to Read News Sites from bottom [Old News] To Top [Latest News]?

Sites Im referring to:

https://www.abc.net.au/news/justin

or 

https://www.tradingview.com/news/world/?area=OCN

Is there also the ability to monitor and read new headlines without having to read the rest that it has done?

thanks

Paul

**Comment by ken107** [comment]
Sorry, the extension is designed to read aloud any website, not just news sites.  And even news sites vary widely in the way they render their content, such that there isn't a universal way to detect and know the page structure and make intelligent ordering decision.  So we can only read from top to bottom.

Monitoring content is also difficult to do, again because we're using a very generic algorithm to handle ALL the possible websites on the internet.  Any custom handling we do for a specific website is not going to work for others.

**Comment by icemanmelb** [comment]
Thank you for your quick respond. Sounds like it's going to be a bit tricky to assess.

By the way, if your software can read text from top to bottom, wouldn't it just be simple to switch it around? Bottom to top? especially on sites that are working already?

thanks

Paul

**Reply by ken107** [reply]
Yes, we probably can read it bottom to top, paragraph by paragraph.  That may work out for a few sites, but for vast majority of sites it will read articles backward, reading footer before header, etc.  The functionality will have very limited applicability.
